pagerduty/util.go

```diff
@@ -118,13 +118,13 @@ func validateCantBeBlankOrNotPrintableChars(v interface{}, p cty.Path) diag.Diag
 	var diags diag.Diagnostics
 
 	value := v.(string)
-	matcher := regexp.MustCompile(`^$|^[ ]+$|[/\\<>&]`)
```

Comment:
This is inconsistent with the function name. '\\', '/', '&', '<', '>' are not non-printable charaters. We need two separate validation functions, one for validateCantBeBlankOrNotPrintableChars and one for validateCantBeBlankOrNotPrintableOrSpecialChars, the former would be the applicable to service and the latter to escalation policy